src/DevYeah.LMS/DevYeah.LMS.Web/app/src/features/common/Navbar.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import logo from '../../logo.png';
+
+export default function Navbar() {
+  return (
+    <nav className="navbar navbar-light bg-light">
+      <div className="container">
+        <div>
+          <a className="navbar-brand font-weight-bold" href="#">
+            <img src={logo} width="30" height="30" className="d-inline-block align-top" alt="" />
```

Comment:
we prefer to use css, not hardcoding the size, in this case, you may give it an id (like logo), then define css `#logo {xxxxx}`